networking/synthetic/stac-test.py

```diff
@@ -86,6 +86,13 @@ def parse_args():
                         dest = 'test_consumer',
                         help = 'OpenShift node IP for running stac-N1 consumer')
 
+    parser.add_argument('-cores',
+                        '--core-ids',
```

Comment:
two options(short form or detailed) for passing variables like '-p'(short form) or '--producer'. Here short-form (-cores) also i kept a bit detailed just for adding clarity.